ValheimServer/ValheimServer/ViewModels/MainWindowViewModel.cs

```diff
@@ -9,5 +9,11 @@ namespace ValheimServer.ViewModels
 {
     public class MainWindowViewModel
     {
+        private string stmBttnText = "Update SteamCmd";
```

Comment:
I have three changes that need to be done before i approve this.
This is stuff there's no way you knew, but i might as well teach you.
Also i never defined a coding standard. So its my fault haha

1. Dont abbreviate words unless its a universally known abbreviation.
`stmBttnText` -> `steamButtonText`

2. private members should start with a underscore _ and be camelCase.
`private string _steamButtonText = "Update SteamCmd`

3. public properties should be PascalCase.
`public string SteamButtonText`

I left Cmd abbreviated bc its a well known abbreviation.